llvm/lib/Target/AMDGPU/AsmParser/AMDGPUAsmParser.cpp

```diff
@@ -5918,6 +5851,9 @@ bool AMDGPUAsmParser::ParseDirective(AsmToken DirectiveID) {
   if (IDVal == PALMD::AssemblerDirective)
     return ParseDirectivePALMetadata();
 
+  if (IDVal == ".amdgcn_code_object_version")
```

Comment:
Why .amdgcn and not .amdhsa? Don't the other directives use an amdhsa prefix?

Reply:
Agreed, should be amdhsa_code_object_version (see my previous comment). It would be good to fix the LLVM IR module flag to be amdhsa_code_object_version as well for consistency.